### PR TITLE
feat(blog): Mediacurrent case-study 

### PR DESCRIPTION
**Pull request by caitlinthefirst**
## Description
Mediacurrent case study for blog.

Note, this has already been reviewed by [Linda](@lindaleebumblebee), but typographical edits/suggestions are welcome.

### CODE REVIEW

**Comment by caitlinthefirst** [comment]
Note, this has already been reviewed by [Linda](@lindaleebumblebee), but typographical edits/suggestions are welcome.

File: docs/blog/2019-06-28-component-driven-drupal-development/index.md [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+
+## Component-Driven Development for Complex Sites
+
+One of the things that excited Ben about working at Mediacurrent was their component-driven approach to building client sites. Rather than creating libraries of page templates, web page designs were broken down into their component parts which were then built completely outside of Drupal in HTML, CSS, and JavaScript. Meanwhile, the client’s style guide would be generated using [KSS-node](https://github.com/kss-node/kss-node) which could then be incorporated into a basic Drupal structure. Once that structure was in place, individual component files were imported wherever they were needed.
+
+<Pullquote>
+“I thought this was an awesome workflow. It gave us a single source of truth for each component. The same template was used by both front-end and back-end developers. We could reuse that same component anywhere it was needed, and it let the front-ender's focus more on front-end specific concerns like semantics and richer interactivity and accessibility.”
```
Comment (marcysutton) [inline docs/blog/2019-06-28-component-driven-drupal-development/index.md]:
```suggestion
“I thought this was an awesome workflow. It gave us a single source of truth for each component. The same template was used by both front-end and back-end developers. We could reuse that same component anywhere it was needed, and it let the front-enders focus more on front-end specific concerns like semantics and richer interactivity and accessibility.”
```

**Comment by caitlinthefirst** [comment]
Thanks, Marcy! Added your edit.

**Comment by marcysutton** [comment]
Merged! Thanks @caitlinthefirst!

**Pull request merged**